artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/qourum/SharedNothingBackupQuorum.java

```diff
@@ -297,7 +300,7 @@ private boolean isLiveDown() {
             quorumManager.vote(quorumVote);
 
             try {
-               quorumVote.await(LATCH_TIMEOUT, TimeUnit.SECONDS);
+               quorumVote.await(quorumVoteWait, TimeUnit.SECONDS);
             } catch (InterruptedException interruption) {
                // No-op. The best the quorum can do now is to return the latest number it has
```

Comment:
Please add some trace logs before the `await` and on the catch scope to log that the awaut has been interrupted